contrib/storage_plugin/storagectl.py

```diff
@@ -216,22 +244,23 @@ def main():
     config_del_parser.add_argument("name")
     config_del_parser.set_defaults(func=delete_secret, secret_name="storage-config")
 
-    # ./storagectl.py user ...
-    user_parser = subparsers.add_parser("user", description="Manage user", formatter_class=argparse.RawDescriptionHelpFormatter)
-    user_subparsers = user_parser.add_subparsers(help="Manage user")
-    # ./storagectl.py user set USER_NAME SERVER_NAME_1 [SERVER_NAME_2 ...]
-    user_set_default_parser = user_subparsers.add_parser("set")
-    user_set_default_parser.add_argument("name")
-    user_set_default_parser.add_argument("servers", nargs="+", help="")
-    user_set_default_parser.set_defaults(func=user_set)
-    # ./storagectl.py user list [-n USER_NAME_1, USER_NAME_2 ...]
-    user_list_parser = user_subparsers.add_parser("list")
-    user_list_parser.add_argument("-n", "--name", dest="name", nargs="+", help="filter result by names")
-    user_list_parser.set_defaults(func=show_secret, secret_name="storage-user")
-    # ./storagectl.py user delete USER_NAME
-    user_del_parser = user_subparsers.add_parser("delete")
-    user_del_parser.add_argument("name")
-    user_del_parser.set_defaults(func=delete_secret, secret_name="storage-user")
+    # ./storagectl.py groupsc add|delete|list
+    groupsc_parser = subparsers.add_parser("groupsc", description="Manage group storage config", formatter_class=argparse.RawDescriptionHelpFormatter)
+    groupsc_subparsers = groupsc_parser.add_subparsers(help="Manage group storage config")
+    # ./storagectl.py groupsc add GROUP_NAME STORAGE_CONFIG_NAME
+    groupsc_add_parser = groupsc_subparsers.add_parser("add")
```

Comment:
Sure. Actually I suggest later, we only manage storage through rest-api because as storage is part of the PAI, we don't need to manage it in two ways.